keg_elements/db/mixins.py

```diff
@@ -378,3 +379,45 @@ def sqla_before_delete_event(mapper, connection, target):
 
 sa.event.listen(SoftDeleteMixin, 'before_delete', SoftDeleteMixin.sqla_before_delete_event,
                 propagate=True)
+
+
+class LookupMixin(DefaultMixin):
+    label = sa.Column(sa.Unicode(255), nullable=False, unique=True)
+    is_active = sa.Column(sa.Boolean, nullable=False, default=True)
```

Comment:
IMO, this is what `SoftDeleteMixin` is for. You can track the `deleted_utc` that way. `NULL` means active in that case.

If you don't want to inherit from `SoftDeleteMixin` because you don't want the other behaviour, but a datetime is just as valid as the `Boolean` and gives you more detail.